src/syscheckd/db/include/fimDBHelpersUTInterface.hpp

```diff
@@ -24,48 +24,63 @@ namespace FIMDBHelper
                             fim_sync_callback_t sync_callback, logging_callback_t logCallback,
                             std::shared_ptr<DBSync>handler_DBSync, std::shared_ptr<RemoteSync>handler_RSync)
     {
-        FIMDBHelpersUTInterface::initDB(sync_interval, file_limit, sync_callback, logCallback, handler_DBSync, handler_RSync);
+        FIMDBHelpersMock::getInstance().initDB(sync_interval, file_limit, sync_callback, logCallback, handler_DBSync, handler_RSync);
```

Comment:
In the include folder, only the definitions of the interfaces exposed by the library should only have.
if this is just code for unit test, move it to the test folder, to keep the order of tests vs functional code.

Reply:
This issue will be addressed in the following [issue](https://github.com/wazuh/wazuh/issues/11332). The changes on the interface and the approach to testing will also be dealt with in the issue previously mentioned.